cli/tests/unit/os_test.ts

```diff
@@ -186,14 +186,14 @@ unitTest({ perms: { env: false } }, function releasePerm(): void {
   assert(caughtError);
 });
 
-unitTest({ perms: { env: true } }, function freemem(): void {
-  assertNotEquals(Deno.freemem(), "");
+unitTest({ perms: { env: true } }, function memoryInfo(): void {
+  assertNotEquals(Deno.memoryInfo(), {});
```

Comment:
I honestly can't figure out whether it is right check every field. Because `memory_info` function return an empty object when an error occurs. Otherwise it return a `MemoryInfo` object and I expect its fields are properly set. Or they can be `undefined`? I expect Rust code doesn't set fields to undefined but I'm honestly not sure.

Reply:
Asserting `n >= 0` should be fine.

In the _exceptional_ case that it returns an error in rust; an exception should be thrown in the JavaScript runtime as-well (e.g return an op-error).